_posts/2018-12-20-so-long-and-thanks.md

```diff
@@ -0,0 +1,131 @@
+underwriter. As weeks passed by and I got more comfortable at the job, 
+I got better about leaving aside that feeling and asking the question anyways - 
+but I do wish that I had done it sooner.
+
+If I had to do it all over again, I would've asked about five times more questions than I did
+regardless of how basic or obvious the answers seemed.
+Funny thing is, I've heard this advice over and over again
+and yet it fell on deaf ears for me until I went through the experience myself. But I know now
+for the future that I might end up a bit annoying sometimes but at least I'll learn a lot!
+
+# Always in Flux
+A rather unique experience I had during my time at Forward Financing was quickly adjusting
+to the constant change. From when I joined in July to a couple months later in
+December, the company went through a few big changes. We had (and still have) a shortage
+of product managers which taught me how important they are in keeping the
+software development lifecycle (SDLC) running smoothly especially with scoping out tasks and minimizing
+the friction between stakeholders and engineers. 
+
+I would like to start of by talking about a rather significant change in our engineering process where our
+sprint of three weeks + a few days reserved for what we call slack time - a short period where the team
+works on tech debt and projects that are not of immediate business concern -
+was changed to two weeks with no slack time. However, I feel that this did not pan
+out too well due to these reasons:
+* Meeting durations were not modified to reflect the reduction in sprint duration
+which meant that on the days of those meetings, productivity was hindered due to those
+meetings acting as distractions.
+* In our workflow, a lot of the tasks require collaboration from multiple teams.
+Sometimes the back-and-forth between coworkers working on a feature together takes time 
+and with unavoidable external factors (eg. QA team backlogged, Heroku being down),
+engineers could lose a day or two of productive work. Therefore, the impact of a lost day
+in a two-week sprint compared to a three-week sprint proves to be much more significant.
+* Due to the prior points and the lack of slack time to work on tech debt, the non-immediate
+yet non-trivial tech debt tasks were piling up. Ideally, there was to be a portion of
+time during the sprint dedicated towards tackling technical debt; however, this did not work out
+due to the above reducing the time allotted for prioritized feature work, leaving no room for it.
+
+These are just some of the observations I made during this slightly chaotic time - others
+might have completely different interpretations and thoughts of the matter.
+As of the time of writing this post, we have reverted back to a three-week sprint with a week's
+worth of slack time. From this, I learned that processes can and will change, but they don't necessarily
```

Comment:
"I learned that processes can and will change, but they don't necessarily need to be permanent and should be critically assessed to see if they work out or not."

YES! This is a great takeaway. Everything that we do, including our own work process, is iterative. I think if you remember this, you can provide tighter focus on how we try to work out our sprint mechanics issues.